src/components/navigation/navigation.component.html

```diff
@@ -0,0 +1,7 @@
+<nav class="navbar-default navbar-static-side-dark tree" role="navigation">
```

Comment:
Remove these outdated classes, as previously requested. (We do not support a dark theme, apart from anything.)

Reply:
navbar-default and navbar-static-side-dark removed.
Tree class still needed to allow correct styling in MF. Without it, top-level classes will have height 56px instead of the correct 40px.